sechub-doc/src/docs/asciidoc/documents/shared/configuration/sechub_config.adoc

```diff
@@ -131,14 +131,18 @@ include::sechub_config_example11_license_scan_and_code_scan_with_sources_data_re
 ==== Secret scan
 `secretScan` defines the settings for scanning for secrets in your source code.
 
+NOTE: The SecHub client automatically adds every file of the defined source directories. (Not only known programming language file extensions like with `codeScan`)
+
+TIP: SecHub Client: With `-addScmHistory` option, SCM directories like `.git` are also uploaded because secrets could be in the SCM history from previous commits.
```

Comment:
```suggestion
TIP: SecHub Client: With the `-addScmHistory` option SCM directories like `.git` are also uploaded, because secrets could be in the SCM history of previous commits.
```

Reply:
:+1:  changed